src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/snippet/Action.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package jdk.javadoc.internal.doclets.toolkit.taglets.snippet;
+
+/**
+ * An action described by markup. Such an action is typically a compound built
+ * from primitive operations of {@link StyledText}.
+ *
+ * <p><b>This is NOT part of any supported API.
+ * If you write code that depends on this, you do so at your own risk.
+ * This code and its internal interfaces are subject to change or
+ * deletion without notice.</b>
+ */
+// FIXME: uncomment /* sealed */ when minimum boot version >= 17
```

Comment:
Do we still have to wait? Currently `java.lang.reflect.Executable` and `java.lang.constant.ClassDesc` are already sealed. I wonder if uncommenting here is fine.

Reply:
Unlike java.base, which the types you mentioned belong to, jdk.javadoc is built using boot JDK. If the JDK being built is of version N, then boot JDK is of version N-1 or N-2. Typically, the version of boot JDK in the mainline is bumped in a few weeks time after GA has been released.

If I uncommented those FIXMEs now, I would see this:
```
Compiling 241 files for BUILD_jdk.javadoc.interim
src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/snippet/Action.java:38: error: sealed classes are a preview feature and are disabled by default.
public sealed interface Action permits AddStyle, Bookmark, Replace {
       ^
  (use --enable-preview to enable sealed classes)
src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/snippet/Action.java:38: error: '{' expected
public sealed interface Action permits AddStyle, Bookmark, Replace {
                              ^
2 errors
```

Currently in the mainline, java.base is built by JDK 18 and jdk.javadoc is built by JDK 16. The latter is because JDK 17 has not been released yet.